openedx/adg/lms/courseware_override/tests/test_views.py

```diff
@@ -0,0 +1,130 @@
+"""
+Tests for views.py file of courseware_override
+"""
+from datetime import datetime, timedelta
+
+import mock
+import pytest
+
+from common.djangoapps.student.roles import CourseInstructorRole
+from common.djangoapps.student.tests.factories import UserFactory
+from openedx.adg.lms.applications.tests.factories import MultilingualCourseFactory, MultilingualCourseGroupFactory
+from openedx.adg.lms.courseware_override.views import (
+    get_all_course_instructors,
+    get_all_languages_for_course,
+    get_course_enrollment_count
+)
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+
+START_DATE = datetime.today() - timedelta(days=365)
+END_DATE = datetime.today() + timedelta(days=365)
+DUMMY_IMAGE_URLS = {'small': 'small_dummy_url', 'medium': 'large_dummy_url', 'large': 'large_dummy_url'}
```

Comment:
Can we move these constants to a separate file named constants.py inside tests folder